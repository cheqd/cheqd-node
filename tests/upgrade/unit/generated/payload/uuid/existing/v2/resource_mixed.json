{
    "resource": {
        "data": "eyJhdHRyIjpbIm5hbWUiLCJhZ2UiXX0="
    },
    "metadata": {
        "collectionId": "C1594E93-dEd6-4244-9CC4-74af38dab09d",
        "id": "031ce9bf-2960-4936-9aa4-6261a6c72a46",
        "name": "Test Resource Name",
        "version": "",
        "resourceType": "CL-Schema",
        "alsoKnownAs": [
            {
                "uri": "https://example.com/alternative-uri",
                "description": ""
            },
            {
                "uri": "https://example.com/alternative-uri",
                "description": "Alternative URI description"
            }
        ],
        "mediaType": "application/json",
        "created": "2021-01-01T00:00:00Z",
        "checksum": "AKAIW1uO3jl8+fhEMe2Q815vZ+rTztzJmi40yp5JPt8=",
        "previousVersionId": "",
        "nextVersionId": ""
    }
}