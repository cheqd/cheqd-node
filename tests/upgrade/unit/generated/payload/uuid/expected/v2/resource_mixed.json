{
    "resource": {
        "data": "eyJhdHRyIjpbIm5hbWUiLCJhZ2UiXX0="
    },
    "metadata": {
        "collectionId": "a35d7486-1749-55bf-96d4-683958a9a884",
        "id": "031ce9bf-2960-4936-9aa4-6261a6c72a46",
        "name": "Test Resource Name",
        "version": "",
        "resourceType": "CL-Schema",
        "alsoKnownAs": [
            {
                "uri": "https://example.com/alternative-uri",
                "description": ""
            },
            {
                "uri": "https://example.com/alternative-uri",
                "description": "Alternative URI description"
            }
        ],
        "mediaType": "application/json",
        "created": "2021-01-01T00:00:00Z",
        "checksum": "AKAIW1uO3jl8+fhEMe2Q815vZ+rTztzJmi40yp5JPt8=",
        "previousVersionId": "",
        "nextVersionId": ""
    }
}